examples/QA-example/main.go

```diff
@@ -21,6 +22,12 @@ var dimensions = 1536
 var numDocsInReq = 5
 
 func main() {
+	// load .env with joho/godotenv
+	err := godotenv.Load()
+	if err != nil {
+		log.Println("Error loading .env file")
+	}
```

Comment:
```suggestion
	if err := godotenv.Load(); err != nil {
		log.Println("issue .env file:", err)
	}
```

can you reflect these in the other examples? How does this end up if .env doesn't exist?